src/app-root.ts

```diff
@@ -90,10 +90,6 @@ export class AppRoot extends LitElement {
     this.searchQuery = this.baseQueryField.value;
     this.collectionBrowser.searchType = this.searchType;
 
-    // Need to request a search in cases where the search type was previously changed but not the query.
-    // If the query was changed above, this should be a no-op.
-    this.collectionBrowser.requestSearch();
-
```

Comment:
Removed Offshoot's reliance on this, so bringing the demo app in line with the same flow.